terraform-provider/examples/resources/cosmo_subgraph/resource.tf

```diff
@@ -0,0 +1,15 @@
+terraform {
+  required_providers {
+    cosmo = {
+      source  = "terraform.local/wundergraph/cosmo"
+      version = "0.0.1"
+    }
+  }
+}
+
+resource "cosmo_subgraph" "test" {
+  base_subgraph_name = var.base_subgraph_name
```

Comment:
We decided to remove `base_subgraph_name` because it is only relevant for feature flags.